kobra.py

```diff
@@ -282,6 +297,11 @@ def draw_grid():
     score = BIG_FONT.render(f"{len(snake.tail)}", True, SCORE_COLOR)
     arena.blit(score, score_rect)
 
+    # Show high score
+    high_score_text = SMALL_FONT.render(f"High: {high_score}", True, SCORE_COLOR)
+    high_score_rect = high_score_text.get_rect(center=(WIDTH - 100, HEIGHT/20 + HEIGHT/30))
+    arena.blit(high_score_text, high_score_rect)
```

Comment:
This strategy for placement isn't very stable, and can result in issues when changing font size, window size or the length of the score increasing (I'm also aware that this is being used on other places in the code, will setup an issue later).

What do you think about using the following approach?

```suggestion
    high_score_text = SMALL_FONT.render(f"High: {high_score}", True, SCORE_COLOR)
    high_score_rect = high_score_text.get_rect(topright=(WIDTH - 32, 32))
    arena.blit(high_score_text, high_score_rect)
```